docs/UsersGuide/source/Quickstart.rst

```diff
@@ -97,7 +97,7 @@ For a detailed explanation of how to build and run the SRW App on any supported
 
       .. code-block:: console
 
-         ./generate_FV3LAM_wflow.sh
+         python generate_FV3LAM_wflow.py
```

Comment:
This could be just `./generate_FV3LAM_wflow.py` as it has the executable file premission.

Reply:
@natalie-perlin I can only speak of tools I added `generate_FVLAM_wflow.py` and `config_utils.py` which are facing the user both have the executable permission. Other python files mostly provide support for workflow generation and do not have the executable permission.